tailwind.config.js

```diff
@@ -78,7 +78,7 @@ module.exports = {
     },
     blur: {
       0: "0",
-      none: "0",
+      none: "",
```

Comment:
Why is this needed? It feels interesting that you wouldn't be able to get this behavior with `blur-0`.

Reply:
This is an edit by ShadCN -- I had to carefully go through my config and make sure my color editions weren't touched, for example. I'm not sure why they made this decision, but I don't really want to change it until we are finished messing with their components.